.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/venv/bin/python"
+}
```

Comment:
I'm not sure but i think this belongs in the `.gitignore`?

Reply:
👍

Reply:
@Awinja-Andela this file should be gitignored.

Reply:
Ignored.
 but i cant seem to find it anywhere to do an git rm